kolena/_experimental/dataset/dataset.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2021-2023 Kolena Inc.
```

Comment:
Any suggestion better than `dataset.dataset`?

Reply:
can we add `register_dataset()` and `fetch_dataset()` to `__init__.py` so we can import them by `from kolena._experimental.dataset import register_dataset, fetch_dataset`?

Reply:
updated as suggested